recipes/pycoast/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pycoast" %}
+{% set version = "1.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "3359b0cc977b13b4b96628a7c2884e85548a08e57220d78734632f914ed1246c" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - pyshp
+    - numpy
+    - pyproj
+    - pillow
+    - six
+    - aggdraw
+  run:
+    - python
+    - pyshp
+    - numpy
+    - pyproj
+    - pillow
+    - six
+    - aggdraw
+
+test:
+  imports:
+    - pycoast
+    - pycoast.tests
+
+about:
+  home: https://github.com/pytroll/pycoast
+  license: GPLv3
+  license_family: GPL3
+  license_file: LICENSE.txt
+  summary: Writing of coastlines, borders and rivers to images in Python
+  description: Writing of coastlines, borders and rivers to images in Python
+  doc_url: http://pycoast.readthedocs.io
+  dev_url: https://github.com/pytroll/pycoast
+
+extra:
+  recipe-maintainers:
```

Comment:
Can you confirm that all of the listed co-maintainers have agreed to be included?

Reply:
It was mentioned in another of the satpy dependency PRs that we are waiting on explicit permission from these co-maintainers. @adybbroe @pnuu @mraspaud

Reply:
Yes, I answered that I was ok with being a maintainer.

Reply:
Same here, I'm ok with being a maintainer.